src/core/js/libraries/jquery.a11y.js

```diff
@@ -1067,6 +1068,10 @@
                 $hideable = $(domSelectors.hideableElements).filter(domFilters.globalTabIndexElementFilter);
             }
 
+            var $branch = this.add(this.parents());
+            var $siblings = $branch.siblings().filter(domFilters.globalTabIndexElementFilter);
```

Comment:
When we spoke briefly about this, you said you may want to switch to `nextAll()`.

Reply:
this was a different bit of code i was meaning. i misunderstood you.
this bit gets all of the elements which aren't associated with the branch your element exists on. 
such that all other elements other than the branch from root to tip that you need can become excluded to the screen reader.

Reply:
@oliverfoster was struggling to get my head around this, but I think the key is to realise that when the ancestors and their siblings are made aria-hidden then by virtue _their_ descendants are hidden. Hope I've got that right.

Reply:
Exactly. This happens through each parent towards the document. The parent's siblings are made aria hidden. As if picking off all the branches down a twig except the one you need to be available.